apps/client-e2e/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
find out what the `tsconfig` do.
No need to make any changes to this file

Reply:
Added to things to learn list